math/smatrix/test/testInversion.cxx

```diff
@@ -14,9 +14,7 @@
 #include "TStopwatch.h"
 
 // matrix size
-#ifndef N
-#define N 5
-#endif
+constexpr unsigned int N = 5;
```

Comment:
Did I see this land in master independently?

Reply:
No, this landed in roottest. The `testInversion.cxx` code is duplicated in root and roottest, with only the code in roottest being actually used. That's maybe not the best situation, but I tried to keep the code syncronized for now. Maybe the code in root should be removed at some point.